lib/system-info.js

```diff
@@ -0,0 +1,28 @@
+const os = require('os');
+
+const packageJson = require('../package.json');
+
+function getSystemInfo() {
+  let systemInfo = {};
+
+  try {
+    systemInfo = {
+      agentVersion: packageJson.version || '',
```

Comment:
Would it be worth lifting this out of the `try`, so we at least get something when there's an error (and can tell people to upgrade)?